tests/openid/connect/core/endpoints/test_openid_connect_params_handling.py

```diff
@@ -28,7 +28,8 @@ def setUp(self):
             'redirect_uri': 'https://a.b/cb',
             'response_type': 'code',
             'client_id': 'abcdef',
-            'scope': 'hello openid'
+            'scope': 'hello openid',
+            'ui_locales': 'en-US'
```

Comment:
By adding this line the `test_authorization_endpoint_handles_prompt` will fail without the patch.

I also added a line to `def test_oidc_params_preservation(self):` to verify `ui_locales` is parsed.